recipes/lark-parser/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "lark-parser" %}
+{% set version = "0.6.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 43d623edae6f16db038d29f8586406002f2bea63c0363dc9acbedac27de7040e
+
+build:
```

Comment:
If this pkg runs on Python 2 and 3, you can use `noarch: python`: https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#architecture-independent-packages